docs/develop/mcp/launch_mcp_server.md

```diff
@@ -81,6 +85,12 @@ The MCP server is designed as an optional component that complements the RAGFlow
         - --mcp-script-path=/ragflow/mcp/server/server.py
         - --mcp-mode=self-host
         - --mcp-host-api-key=ragflow-xxxxxxx
+        # Optional transport flags for MCP (customize if needed).
+        # Host mode need to combined with --no-transport-streamable-http-enabled flag, namely, host+streamable-http is not supported yet.
```

Comment:
```suggestion
        # If you set `mcp-mode` to `host`, you must add the --no-transport-streamable-http-enabled flag, because the streamable-HTTP transport is not yet supported in host mode.
```